src/components/PageContent/PageContent.astro

```diff
@@ -10,6 +10,7 @@ const title = content.title
 	.replace('@astrojs/', '<span class="scope">@astrojs/</span><wbr>')
 	.replaceAll('-', '&#8288;-&#8288;');
 const description = content.description;
+const suppressTitle = (content as any).suppressTitle;
```

Comment:
Is it safe to use `as any` here? Isn't it possible to update the underlying type of `content` to add this field?